main.cpp

```diff
@@ -419,26 +397,12 @@ struct Envelope
 
 bool Envelope::willOutputCV(bool powerOn, bool gateHigh)
 {
-    if (powerOn == true && gateHigh == true)
-    {
-        return true;
-    }
-    else
-    {
-        return false;
-    }
+    return (powerOn == true && gateHigh == true);
```

Comment:
You can simplify this further:
```
return powerOn && gateHigh;
```
they're booleans, so you don't need to compare them with `true`.  The only reason to do that would be to improve the readability.